### ISSUE TITLE
Installation errors

### ISSUE THREAD

**Issue by weiss-d** [issadarkthing/gomu#4]
Hi there! This app looks very interesting, but I cannot install it (Pop!_OS 20.04) getting these errors:

```bash
$ go get -u github.com/issadarkthing/gomu
# github.com/rivo/tview
go/src/github.com/rivo/tview/box.go:343:63: cannot convert b.borderAttributes (type tcell.AttrMask) to type tcell.Style
go/src/github.com/rivo/tview/table.go:331:108: cannot convert attributes (type tcell.AttrMask) to type tcell.Style
go/src/github.com/rivo/tview/table.go:888:150: cannot convert cell.Attributes (type tcell.AttrMask) to type tcell.Style
go/src/github.com/rivo/tview/table.go:957:63: cannot convert a (type tcell.AttrMask) to type tcell.Style
go/src/github.com/rivo/tview/table.go:1020:24: invalid operation: t.selectedStyle != 0 (mismatched types tcell.Style and int)
```

**Comment by Prajwal-Koirala** [comment]
Same here.

**Comment by issadarkthing** [comment]
This issue seems like it have been fixed https://github.com/rivo/tview/issues/506#issuecomment-711160092

**Comment by weiss-d** [comment]
> This issue seems like it has been fixed [rivo/tview#506 (comment)](https://github.com/rivo/tview/issues/506#issuecomment-711160092)

Cannot find the solution there 🤯

**Comment by issadarkthing** [comment]
Try reinstall. I have tried it in VM, it works just fine for me.

**Comment by weiss-d** [comment]
> Try reinstall. I have tried it in VM, it works just fine for me.

Yeah, that works!
Although, the istallation procedure is a little bit obscure for those who are not into Go gevelopment. I mean, the command hangs for 5-10 seconds and then finishes without any output. And then only by intuition I found `~/go/bin` folder and `gomu` binary inside :)

Anyway, accept my congratulations 'cause this player feels much more clean and intuitive than every console player that I've tried (cmus, ncmcpp+mpd etc.).
It would be great if it'll have CUE support one day!
